README.md

```diff
@@ -47,7 +47,9 @@ the latest compatible version each time that the MAJOR.MINOR version of `rubocop
 is updated.
 
 ## Custom Cops
+1. [RspecDotNotSelfDot](https://github.com/ezcater/ezcater_rubocop/blob/master/lib/rubocop/cop/ezcater/rspec_dot_not_self_dot.rb) - Enforce ".<class method>" instead of "self.<class method>" for example group description.
```

Comment:
These links, on the other hand, should link to master to ensure they always link to the current version.... 👍